suite/tests/CMakeLists.txt

```diff
@@ -1706,6 +1706,7 @@ if (CLIENT_INTERFACE)
   include_directories("${PROJECT_BINARY_DIR}/suite/tests/annotations")
 
   tobuild_ci(client.call-retarget client-interface/call-retarget.c "" "" "")
+  tobuild_ci(int64_overrides client-interface/int64_overrides.c "" "" "")
```

Comment:
Please match all of the peer tests: they have a `client.` prefix, so `client.int64-overrides`.

Reply:
done